Intersect.Client/Entities/Entity.cs

```diff
@@ -989,7 +1054,7 @@ public virtual void Draw()
                 return;
             }
 
-            var d = (Dir + (Options.Instance.Sprites.Directions - 1)) % Options.Instance.Sprites.Directions;
+            var d = mEntitySpriteDirections[(int)Dir];
```

Comment:
Make the method `static int PickSpriteRow(Direction direction)` that uses a switch to return the integer.

In this one method, casting from Direction to int makes sense since the return value is no longer a direction, and using Direction is more expressive with less documentation.

Reply:
![imagen](https://user-images.githubusercontent.com/17498701/217347029-18049419-1171-4e62-98a2-2b6a596fd698.png)

![imagen](https://user-images.githubusercontent.com/17498701/217347065-0192d586-a30a-4f28-a773-1c6d2975a621.png)

Reply:
Combine all of the cases that result in the same value, and sort the cases alphabetically.

Reply:
![imagen](https://user-images.githubusercontent.com/17498701/217389922-50ca5394-6a4b-4a7a-ac9a-d129c056d626.png)